### DISCUSSION TITLE
Possible to change the line padding?

### DISCUSSION THREAD

**Discussion by spaceexperiment** [zed-industries/zed#6508] [Feature Requests]
Ultimatly I would like to reduce the space between lines, is this currently possible?

**Comment by fdionisi** [comment]
@spaceexperiment, if I understand correctly, you are talking about adjusting the line height which is not currently possible. But you can 👍 to [this issue](https://github.com/zed-industries/community/issues/304), so the team is informed and can prioritise the feature. 🙏

**Comment by JosephTLyons** [comment]
Line height is now configurable.

- https://github.com/zed-industries/zed/issues/5590
